app/models/premium.rb

```diff
@@ -0,0 +1,39 @@
+module Premium
+  MONTH_AMOUNT_IN_CENTS = 9_99
+  YEAR_AMOUNT_IN_CENTS = 99_99
+  LIFETIME_AMOUNT_IN_CENTS = 49_900
+
+  MONTH_AMOUNT_IN_DOLLARS = MONTH_AMOUNT_IN_CENTS / 100.0
+  YEAR_AMOUNT_IN_DOLLARS = YEAR_AMOUNT_IN_CENTS / 100.0
+  LIFETIME_AMOUNT_IN_DOLLARS = LIFETIME_AMOUNT_IN_CENTS / 100.0
+
+  def self.amount_in_cents_from_interval(interval)
+    case interval.to_sym
+    when :month
+      MONTH_AMOUNT_IN_CENTS
+    when :monthly
+      MONTH_AMOUNT_IN_CENTS
```

Comment:
The `month/monthly` bit is due to PayPal using both month and monthly for intervals.